prowler/providers/gcp/gcp_provider.py

```diff
@@ -38,13 +38,18 @@ class GcpProvider(Provider):
     # TODO: this is not optional, enforce for all providers
     audit_metadata: Audit_Metadata
 
-    def __init__(self, arguments):
+    def __init__(
```

Comment:
Please add default values and document the class.

Reply:
Done